### PR TITLE
[RC1] Update harness tests for Sepolia

### PR DESCRIPTION
**Pull request by ytham**
- Rebase https://github.com/axiom-crypto/axiom-sdk-client/pull/9 for `rc1` branch.

### CODE REVIEW

File: circuit/js/test/unit/encoder.test.ts [hunk 1]
```diff
@@ -19,8 +19,9 @@ describe("Encoder", () => {
       maxOutputs: AxiomV2CircuitConstant.UserMaxOutputs,
       isAggregation: false,
     }
+    console.log("DEFAULT_METADATA", DEFAULT_METADATA);
     const encoded = encodeAxiomV2CircuitMetadata(DEFAULT_METADATA);
-    const expected = "0x0001000009000100000004010000010080000000000000000000000000000000";
+    const expected = "0x0001000009000100000014030000010080000000000000000000000000000000";
```
Comment (ytham) [inline circuit/js/test/unit/encoder.test.ts:24]: needed to modify this based on the new wider circuit config in tools: https://github.com/axiom-crypto/axiom-tools/commit/d25d64ac9cec57257ec7ebe513828f7e89a6ffb4

File: circuit/js/src/constants.ts [hunk 2]
```diff
@@ -15,9 +15,9 @@ export const COMPUTE_NUM_INSTANCES = USER_COMPUTE_NUM_INSTANCES + SUBQUERY_NUM_I
 
 export const DEFAULT_CIRCUIT_CONFIG = {
```
Comment (ytham) [inline circuit/js/src/constants.ts:16]: had to update these constants as well. i couldn't find any tools constants for the other values; let me know if we need to modify anything else.

File: circuit/js/test/unit/encoder.test.ts (hunk 1 above)
Comment (jonathanpwang) [reply circuit/js/test/unit/encoder.test.ts:24]: This matches https://github.com/axiom-crypto/axiom-eth-working/pull/238

File: circuit/js/src/constants.ts (hunk 2 above)
Comment (jonathanpwang) [reply circuit/js/src/constants.ts:16]: The other ones should stay same and aren't in axiom-tools.
`numLookupBits` should always be `k - 1`, and when the circuit does auto_config, it should select a new `k` optimally.

**Review by jonathanpwang — APPROVED** [review]
LGTM

**Pull request merged**